spec/menu_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'menu'
+require 'formatter'
+
+include Formatter
+
+describe Menu do
+
+  let(:dish_klass){ double(:dish_klass, new: dish) }
+  let(:dish){ double(:dish, name: 'Burger', price: 5.99) }
+
+  let(:file_klass){ double(:file_klass, open: file)}
+  let(:line){ 'Burger,5.99' }
+  let(:file){ double(:file) }
+
+  subject(:menu){ Menu.new(dish_klass, file_klass) }
+
+  before :each do
+    line = 'Burger,5.99'
+    allow(file).to receive(:gets).and_return(line, nil)
+  end
+
+  context '#formatted_dishes' do
+    it 'returns a formatted list of dishes' do
+      name, price = dish.name, pound_str(dish.price)
```

Comment:
Style/ParallelAssignment: Do not use parallel assignment. (https://github.com/bbatsov/ruby-style-guide#parallel-assignment)